src/relay/qnn/op/batch_matmul.cc

```diff
@@ -96,20 +96,42 @@ Expr BatchMatmulFirstTerm(const Expr& quantized_x, const Expr& quantized_y,
 }
 
 Expr BatchMatmulSecondTerm(const Expr& x_quantized_data, const Expr& y_zero_point) {
-  Array<Integer> axes = {2};
-  return Multiply(y_zero_point, Sum(Cast(x_quantized_data, DataType::Int(32)), axes, true, false));
+  if (IsScalar(y_zero_point)) {
```

Comment:
```suggestion
  ICHECK(IsScalar(y_zero_point)) << "Tensor ZP is not supported"
```
Do you think return empty Expr() is better choice? I am afraid it will be not clear the reason of fail when not scalar ZP will be provided.

Reply:
done